### PR TITLE
Fetch and Servlet

### PR DESCRIPTION
**Pull request by glstewart17**
This branch completes week 3 part 1 and part 2.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/OfficeQuoteServlet.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    quotes.add("I'm not supersitious, but I am a little stitious.");
+    quotes.add("The worst thing about prison was the dementors.");
+    quotes.add("I talk a lot. so I've learned to tune myself out.");
+    quotes.add(
+        "Sometimes I'll start a sentence and I don't even know where it's going. "
+            + "I just hope I find it along the way.");
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (atupem) [inline portfolio/src/main/java/com/google/sps/servlets/OfficeQuoteServlet.java:44]: NItpick: make sure to add documentation for this method

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -65,3 +65,42 @@ function showSlide() {
...
+/**
+ * Fetches a random quote from the Office from the server and adds it to the DOM.
+ */
+function getOfficeQuote() {
+  console.log('Fetching a quote from the Office.');
+
+  // The fetch() function returns a Promise because the request is asynchronous.
+  const responsePromise = fetch('/random-office-quote');
+
+  // When the request is complete, pass the response into handleResponse().
```
Comment (atupem) [inline portfolio/src/main/webapp/script.js:78]: What happens if the request fails?

File: portfolio/src/main/java/com/google/sps/servlets/OfficeQuoteServlet.java (hunk 1 above)
Comment (glstewart17) [reply portfolio/src/main/java/com/google/sps/servlets/OfficeQuoteServlet.java:44]: I'll add it in the next pr. Ironically, this file is being removed.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (glstewart17) [reply portfolio/src/main/webapp/script.js:78]: I'll resolve this for the next pr.

**Pull request merged**